charts/newrelic-infrastructure/templates/NOTES.txt

```diff
@@ -20,17 +20,18 @@ You have specified ksm or kubelet integration components as not enabled.
 Those components are needed to have the full experience on NROne kubernetes explorer.
 {{- end }}
 
-{{- if and .Values.controlPlane.enabled (not (include "newrelic.controlPlane.hostNetwork" .)) }}
+{{- if and .Values.controlPlane.enabled (not (include "common.hostNetwork" . | default .Values.controlPlane.hostNetwork)) }}
```

Comment:
![Screenshot 2022-04-07 at 18 48 37](https://user-images.githubusercontent.com/53659978/162255570-7646a711-29d4-4516-92c8-6a40440d7017.png)

Well... This is kind of tricky because `default` reads falsiness so from global and local toggles are going to be overridden by the `values.yaml` where `.Values.controlPlane.hostNetwork` is true.

So virtually is the same as putting directly this:
```suggestion
{{- if and .Values.controlPlane.enabled .Values.controlPlane.hostNetwork }}
```

WDYT?

Reply:
Think that this is for the `NOTES.txt`, not for the logic of any manifest.

Reply:
You were completely right, I don't know what I was thinking when I wrote that (and it seems that I was completely convinced of that).

Solved on [#a1b6f77](https://github.com/newrelic/nri-kubernetes/pull/383/commits/a1b6f77ff8c7384c6873cd52c42aa9ca2068523c)